add tsh is slow entry to FAQ (#29546)
gravitational/teleport: docs/pages/faq.mdx

Make sure that all Teleport components are at least at version 10.3.0. Older versions
don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.

## `tsh` is very slow on Windows, what to do?

> If your host machine is joined to an Active Directory domain, you might find user lookups take a lot longer than you expect because of the number of Active Directory accounts that must be scanned. To fix this issue, you can use environment variables to set default account information for your Teleport user. If you are experiencing long lookup times on Windows, do the following:

Reviewer: - this is a very long line, can we break it up to roughly match the length used in the rest of the page?
- I would end the sentence after "longer than you expect" and start a new sentence that explains why.

Author: Done

> If your host machine is joined to one or more Active Directory domains, you might find user lookups take a lot longer than you expect because of the number of Active Directory accounts that must be scanned in your environment. To fix this issue, you can use environment variables to set default account information for your Teleport user. If you are experiencing long lookup times on Windows, do the following:

Author: Good catch, I was thinking about a domain with many trusts when I wrote that. Didn't realize only one domain could be joined to though. I'll just leave it at `If your host machine is joined to an Active Directory domain` to keep it simple, as one domain with no joins could have enough users to trigger this issue as well as a domain that has many trusts.

- Make sure `tsh` is at least at version 12.2.4.

> - Either set the `TELEPORT_USER` environment variable or set the `--user` flag to the name of your Teleport User.

Reviewer 2: ```suggestion
- Either set the `TELEPORT_USER` environment variable or set the `--user` flag to the name of your Teleport user.
```

> - Either set the `TELEPORT_LOGIN` environment variable or set the `--login` flag to the name of current host user. This should get overridden if you want to SSH to a machine as a different user.

Reviewer 2: ```suggestion
- Either set the `TELEPORT_LOGIN` environment variable or set the `--login` flag to the name of current host user. This setting can be overridden if you open a new SSH session on a machine as a different user.
```

> - Set the `TELEPORT_HOME` environment variable to be the home directory of your current host user + `\.tsh`. So if your home directory is `C:\Users\Me`, you'd set `TELEPORT_HOME` to `C:\Users\Me\.tsh`.

Reviewer 2: ```suggestion
- Set the `TELEPORT_HOME` environment variable to be the home directory of your current host user + `\.tsh`. For example, if your home directory is `C:\Users\Me`, you'd set `TELEPORT_HOME` to `C:\Users\Me\.tsh`.
```

Make sure that all Teleport components are at least at version 10.3.0. Older versions
don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.

## `tsh` is very slow on Windows, what to do?

> This may be because user lookups take a long time in your environment. Make sure `tsh` is at least

Reviewer: I think we've only observed this with AD-joined hosts. We should probably mention that here.

Author: Done

> time in your environment. To fix this, make sure `tsh` is at least at version 12.2.4. Either set

Reviewer 2: It might be nice to add a sentence or two about why this happens and why this is the way to fix it. Something like this maybe (but note that I'm just making the following up as a "best guess"):
If your host machine is joined to one or more Active Directory domains, you might fine user lookups take a longer than you expect because of the number of Active Directory accounts that must be scanned in your environment. To fix this issue, you can use environment variables to preset account information for your Teleport user.  If you are experiencing long lookup times on Windows, do the following:

- Make sure `tsh` is at least at version 12.2.4 or upgrade to the latest version.
- Set `TELEPORT_USER` environment variable or set the `--user` flag to the name of your Teleport user.
- Set `TELEPORT_LOGIN` environment variable or set the `--login` flag to the name of current
host user. This should get overridden if you want to SSH to a machine as a different user.
- Set
`TELEPORT_HOME` environment variable to be the home directory of your current host user + `\.tsh`.
So if your home directory is `C:\Users\Me`, you'd set `TELEPORT_HOME` to `C:\Users\Me\.tsh`.

Author: Done

`TELEPORT_USER` environment variable or set the `--user` flag to the name of your Teleport User.
Either set `TELEPORT_LOGIN` environment variable or set the `--login` flag to the name of current
host user. This should get overridden if you want to SSH to a machine as a different user. Set
`TELEPORT_HOME` environment variable to be the home directory of your current host user + `\.tsh`.
So if your home directory is `C:\Users\Me`, you'd set `TELEPORT_HOME` to `C:\Users\Me\.tsh`.

> You can set these environmental variables globally in Windows so you don't have to set them every

Reviewer 2: ```suggestion
You can set these environmental variables globally in Windows so that you don't have to set them every
```

don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.

## `tsh` is very slow on Windows, what to do?

If your host machine is joined to an Active Directory domain, you might find user lookups take a
lot longer than you expect. The number of Active Directory accounts that must be scanned to
preform a user lookup can cause tsh to hang waiting to get information about the current user.
To fix this issue, you can use environment variables to set default account information for your 
Teleport user. If you are experiencing long lookup times on Windows, do the following:

- Make sure `tsh` is at least at version 12.2.4.
- Either set the `TELEPORT_USER` environment variable or set the `--user` flag to the name of your Teleport user.
- Either set the `TELEPORT_LOGIN` environment variable or set the `--login` flag to the name of current host user. This setting can be overridden if you open a new SSH session on a machine as a different user.
- Set the `TELEPORT_HOME` environment variable to be the home directory of your current host user + `\.tsh`. For example, if your home directory is `C:\Users\Me`, you'd set `TELEPORT_HOME` to `C:\Users\Me\.tsh`.

> You can set these environmental variables globally in Windows so that you don't have to set them every

Reviewer: ```suggestion
You can set these environment variables globally in Windows so that you don't have to set them every
```
